app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="container">
+  <h1><%= @pokemon.name %></h1>
+  <h3>Rate: <%= @pokemon.rate %> / Level: <%= @pokemon.level %></h3>
+  <p>Type: <%= @pokemon.type %> / Location: <%= @pokemon.location %></p>
```

Comment:
will .type work? isn't it .pokemon-type now?

Reply:
Good point! Fixed!